rclpy/test/test_qos_event.py

```diff
@@ -95,21 +93,18 @@ def test_publisher_constructor(self):
 
         # Arg with three callbacks
         callbacks.incompatible_qos = incompatible_qos_callback
-        try:
-            publisher = self.node.create_publisher(
-                EmptyMsg, self.topic_name, 10, event_callbacks=callbacks)
-            self.assertEqual(len(publisher.event_handlers), 3)
-            self.node.destroy_publisher(publisher)
-        except UnsupportedEventTypeError:
```

Comment:
There's another flake8 error because we no longer are using the import for `get_rmw_implementation_identifier`.